python/mxnet/rnn/rnn_cell.py

```diff
@@ -0,0 +1,172 @@
+# coding: utf-8
+"""Definition of various recurrent neural network cells."""
+
+import copy
+
+from .. import symbol
+from .. import ndarray
+from ..base import numeric_types, string_types
+
+class RNNParams(object):
```

Comment:
Suggest comment
"""This class holds the learnable parameters (weights, biases) for the RNN.
New params are added as needed with get()
"""